SyncIntegrationTests/xcodebuild.py

```diff
@@ -31,10 +31,12 @@ def test(self, identifier):
             out = subprocess.check_output(
                 args,
                 cwd=os.path.join(here, os.pardir),
-                stderr=subprocess.STDOUT)
+                stderr=subprocess.STDOUT,
+                universal_newlines=True)
```

Comment:
This makes the resulting log output be rendered as nice text, rather than a stringified bytes object like `b"output\nnot very readable\nwith escaped\nnewlines"`.